string Name
string userID
int32 isExist
int32 FirstRec
string FirstQ
int32 SecondRec
string SecondQ
int32 ThirdRec
string ThirdQ
int32 ForthRec
int32 FifthRec
